src/app/core/basic-datatypes/date/date-range-filter/date-range-filter-panel/date-range-filter-panel.component.ts

```diff
@@ -30,18 +30,39 @@ export const defaultDateFilters: DateRangeFilterConfigOption[] = [
   },
   {
     startOffsets: [{ amount: -1, unit: "weeks" }],
```

Comment:
The default options here have been designed for filtering notes. If we add quarters and others, that list is becoming very long and overwhelming. Instead, I would like to define different default ranges when we use this for the report view

i.e. probably in the `SelectReportComponent` we can create a different object of defaultReportDateFilters, similar to this one but with different ranges (as listed in the issue) and pass that in its template there